### PR TITLE
7.9.2 build

### PR DESCRIPTION
**Pull request by amottier**
This PR should allow a successful build of Bonita 7.9.2 Community Edition.

### CODE REVIEW

File: build-script.sh [hunk 1]
```diff
@@ -2,323 +2,409 @@
...
+# bonita-continuous-delivery-doc: Bonita Enterprise Edition Continuous Delivery module documentation.
+# bonita-custom-page-seed: a project to start building a custom page. Deprecated in favor of UI Designer page + REST API extension.
+# bonita-doc: Bonita documentation.
+# bonita-developer-resources: guidelines for contributing to Bonita, contributor license agreement, code style...
+# bonita-examples: Bonita usage code examples.
+# bonita-ici-doc: Bonita Enterprise Edition AI module documentation.
+# bonita-js-components: automatically downloaded in the build of projects that require it.
+# bonita-migration: migration tool to update a server from a previous Bonita release.
+# bonita-page-authorization-rules: documentation project to provide an example for page mapping authorization rule.
+# bonita-platform: deprecated, now part of bonita-engine repository.
```
Comment (tbouffard) [inline build-script.sh]: ⚠️ to be removed as this is an archived repo

File: build-script.sh [hunk 2]
```diff
@@ -2,311 +2,409 @@
...
+# FIXME: remove when temporary workaround become useless
+STUDIO_P2_URL_INTERNAL_TO_REPLACE=http://repositories.rd.lan/p2/4.10.1
+
 # Test that x server is running. Required to generate Bonita Studio models
 # Can be ignored if Studio is build without the "generate" Maven profile
+# Warning: this requirement prevents to build on Travis CI
 if ! xset q &>/dev/null; then
-    echo "No X server at \$DISPLAY [$DISPLAY]" >&2
-    exit 1
+	echo "No X server at \$DISPLAY [$DISPLAY]" >&2
```
[the remark this answers is not in the document]
Comment (amottier) [reply build-script.sh]: I prefer tabs and tabs win over spaces here: https://softwareengineering.stackexchange.com/questions/57/tabs-versus-spaces-what-is-the-proper-indentation-character-for-everything-in-e
Any proper difftool can ignore spacing character so it should not be a big deal

File: build-script.sh [hunk 3]
```diff
@@ -2,323 +2,409 @@
...
 
+build_maven_wrapper_install_maven_test_skip bonita-web-extensions
+# FIXME: see comments on skiptest function
+build_maven_install_skiptest bonita-web
 build_maven_install_maven_test_skip bonita-portal-js
 
-build_maven_install_maven_test_skip bonita-distrib
+# bonita-web-pages is build using a specific version of UI Designer.
+# Version is defined in https://github.com/bonitasoft/bonita-web-pages/blob/$BONITA_BPM_VERSION/build.gradle
+build_maven_install_skiptest bonita-ui-designer 1.9.53
```
[the remark this answers is not in the document]
Comment (amottier) [reply build-script.sh]: True and it was inconsistent with the build of UI Designer embedded in the Studio that is correctly using the wrapper.

File: build-script.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (tbouffard) [reply build-script.sh]: I won't fight on that, useless debate 😸

**Pull request merged**